server/catalog/models.py

```diff
@@ -20,6 +20,7 @@ class DataSet(models.Model):
     system_message = models.TextField(
         default="You are a sales support agent, and you know everything about a company and their products."
     )
+    products_type = models.CharField(blank=True, max_length=255)
```

Comment:
I wouldn't make it a dataset field - if it's used only by one of the agents, for configuring the validation step at the end. I think that should be a configuration parameter for an agent instance instead.